crates/cxx-qt-lib/src/gui/qimage.rs

```diff
@@ -11,12 +11,100 @@ mod ffi {
     unsafe extern "C++" {
         include!("cxx-qt-lib/qimage.h");
         type QImage = super::QImage;
+        include!("cxx-qt-lib/qsize.h");
+        type QSize = crate::QSize;
+        include!("cxx-qt-lib/qrect.h");
+        type QRect = crate::QRect;
+        include!("cxx-qt-lib/qcolor.h");
+        type QColor = crate::QColor;
+        include!("cxx-qt-lib/qpoint.h");
+        type QPoint = crate::QPoint;
 
         /// Whether the QImage is null.
         ///
         /// This means that the QImage has all parameters set to zero and no allocated data.
         #[rust_name = "is_null"]
         fn isNull(self: &QImage) -> bool;
+
+        /// Returns true if all the colors in the image are shades of gray
+        #[rust_name = "all_gray"]
+        fn allGray(self: &QImage) -> bool;
```

Comment:
personally I would try to keep these methods ordered either
- alphabetically
- in the order they are in the Qt docs

(i think the Qt docs are mainly alphabetical so it's the same either way)